dptb/entrypoints/pdso.py

```
@@ -0,0 +1,129 @@
+from pathlib import Path
+from dptb.postprocess.unified.system import TBSystem
+
+log = logging.getLogger(__name__)
+
+def pdso(
+        INPUT: str,
+        init_model: str = None,
+        structure: str = None,
+        data_dir: str = None,
+        output_dir: str = "./",
+        log_level: int = 20,
```

Ruff (0.14.13)

Unused function argument: `log_level`

(ARG001)